qa/automation/CareerPortal.Tests.Scenarios/1. Search by Filters/1.1. Search by Filters.feature

```diff
@@ -0,0 +1,134 @@
+Feature: 1.1. Search by Filters
+
+Background: 
+    Given User is on https://career.quantori.com website
+    And There are 10 available <positions> on the jobs page
+    And There are 2 available <competence> on the jobs page
+    And There are 2 available <technologies> on the jobs page
+    |Position                                              ||competence       ||technologies|
+    |Junior JavaScript Engineer                            ||Databases        ||SQL         |
+    |Senior/Lead Manual QA Engineer (Life Science projects)||Business Analysis||.NET        |
+    |Senior/Lead Manual QA Engineer (internal projects)    ||                 ||            |
+    |Data Engineer (SQL)                                   ||                 ||            |
+    |Snowflake Data Warehouse Certified Engineer           ||                 ||            |
+    |Senior DWH / ETL Data Engineer                        ||                 ||            |
+    |Data Engineer (Spark)                                 ||                 ||            |
+    |Junior JavaScript Engineer                            ||                 ||            |
+    |Senior QA Automation (Python) Engineer / SDET         ||                 ||            |
+    |Senior Data Engineer                                  ||                 ||            |
+
+
+Scenario: 1.1_01 Checking the function of the search button
+    When User pushes "search" button
+    Then User should be redirected to the new page "https://career.quantori.com/ru/positions"
+    And  User should see default page "Quantori open job positions"
+
+Scenario: 1.1_02 Returning to the general job list from the "Quantori Hot Open Job Positions list"
+    Given User opened https://career.quantori.com/ru/positions/cloud-python-engineer website
+    When At the bottom of the page user push "Back to open job positions" button
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And  User should see default page "Quantori open job positions"
+
+Scenario: 1.1_03 Checking the search by 1 competence filter
+    When User chooses "Databases" competence with list box filter Competence
+    And User pushes "search" button
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And User should see only 2 vacancy "Data Engineer (SQL)" and "Snowflake Data Warehouse Certified Engineer"
+
+Scenario Outline: 1.1_04 Checking the search by 1 technology filter
+    When User chooses "SQL" technology with list box filter Technology
+    And User pushes "search" button
+    Then User should be redirected to the new page https://career.quantori.com/ru/positions
+    And User should see only 8 <Position>
+    
+    Examples: 
```

Comment:
No need in Examples section here, it should be just a table:
And following positions should be visible:
|Position                                              |
                |Junior JavaScript Engineer                            |
                |Senior/Lead Manual QA Engineer (Life Science projects)|
                |Senior/Lead Manual QA Engineer (internal projects)    |
                |Data Engineer (SQL)                                   |
                |Senior DWH / ETL Data Engineer                        |
                |Data Engineer (Spark)                                 |
                |Junior JavaScript Engineer                            |
                |Senior QA Automation (Python) Engineer / SDET         |
                |Senior Data Engineer                                  |